tutorial/tests/test_basic_datatypes.py

```diff
@@ -467,5 +472,47 @@ def reference_string_split_lines(my_string: str) -> list[str]:
 
 def test_string_split_lines(function_to_test):
     my_string = "\n".join(STRINGS)
-    assert function_to_test(my_string) == STRINGS
     assert function_to_test(my_string) == reference_string_split_lines(my_string)
+
+
+INT_SETS = [set(), {1, 2, 3}, {3, -1, 0, 4, 42, 1002}]
+
+
+def reference_sets_of_even_and_odd(my_set: set[int]) -> tuple(set[int]):
```

Comment:
```suggestion
def reference_sets_of_even_and_odd(my_set: set[int]) -> tuple[set[int]]:
```

Reply:
I am sorry you had to look for that error, I was using vscode for a lot of my testing and somehow missed that.